src/react-native-cool-modals/ios/RNCMScreenStack.m

```diff
@@ -236,6 +236,17 @@ - (void)setModalViewControllers:(NSArray<UIViewController *> *)controllers
   NSUInteger changeRootIndex = 0;
   UIViewController *changeRootController = _controller;
   
+  // I noticed that sometimes makeOldClass was not called correctly
+  // Before traversing UIViewControllers I restores the previous class
+  UIViewController *someViewController = changeRootController.presentedViewController;
+  while (someViewController != nil) {
+    UIView *view = someViewController.view.superview.superview;
```

Comment:
I have always been super paranoid about this and personally I'd prefer to break it down with checks against nil instead of calling an property 4 levels deep.
Maybe something like
```
if(someViewController.view != nil && someViewController.view.superview != nil && someViewController.view.superview.superview != nil) {
```

Reply:
@brunobar79 
It might be thing is Java/JS. However, in Objc-C is perfectly fine to make it:
https://medium.com/@andy.nguyen.1993/what-happen-when-we-call-a-method-on-a-nil-object-objective-c-5a0aacd7a240